src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -213,6 +213,9 @@
         - name: --ppg
           type: string
           short-summary: The ID of a PPG.
+        - name: --enable-fips-image
+          type: bool
+          short-summary: Enable FIPS VHD image.
```

Comment:
Can we use 'Use FIPS-enabled OS on agent nodes' for short-summary? @palma21

Reply:
Sure, changed.